packages/terra-core-docs/src/terra-dev-site/doc/pills/example/BasicPill.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PillList, { Pill } from 'terra-pills';
+
+const BasicPill = () => (
+  <PillList
+    ariaLabel="Example of a Basic Pill"
+  >
+    <Pill
```

Comment:
Just to clarify can `<pill>` be used without <pillList>...?? IMO example name makes it confusing since the parent component here is PillList.